source/sass/mixins.scss

```diff
@@ -1,16 +1,16 @@
-@mixin scaleText($base-font-size, $base-line-height: null, $large-font-size: null, $large-line-height: null) {
+  @mixin scaleText($base-font-size, $base-line-height: null, $large-font-size: null, $large-line-height: null) {
   font-size: $base-font-size;
 
   @if ($base-line-height) {
-    line-height: $base-line-height/$base-font-size;
+    line-height: $base-line-height;
```

Comment:
to fix
<img width="410" alt="screen shot 2018-05-01 at 12 14 37 pm" src="https://user-images.githubusercontent.com/2896608/39489178-87110b64-4d39-11e8-8493-aa455b0f23f4.png">